Website/src/rendering/src/assets/css/components/Speakers/FeaturedSpeakers.css

```diff
@@ -0,0 +1,40 @@
+.featured-speakers {
+  @apply bg-cover bg-center bg-no-repeat;
```

Comment:
The div with the `featured-speakers` className does not seem to have a background image. I am not sure this line is needed anymore.